src/lib.rs

```
@@ -39,6 +35,9 @@
         //Save the scad code to a file
         scad_file.write_to_file(String::from("out.scad"));
 
+        # // remove the created file
+        # drop(std::fs::remove_file("out.scad"));
+
```

What is the drop doing here? Also, why would we want to remove the generated scad file?

---

Drops the result from `remove_file` (ignores errors). Running (doc) tests creates the out.scad file and this hidden code deletes it. Maybe it would be better to not write anything to disk in the test anyways

---

Oh interesting. Might be better to not create the file at all, yea. I think you can add `norun` to the top of the markdown code block to ignore that specific test.